tests/functional/ext/typing/typing_broken_callable.py

```diff
@@ -4,7 +4,7 @@
 
 Use 'typing.Callable' instead.
 """
-# pylint: disable=missing-docstring,unsubscriptable-object
+# pylint: disable=missing-docstring,unsubscriptable-object, invalid-name
```

Comment:
```suggestion
# pylint: disable=missing-docstring,unsubscriptable-object,invalid-name
```
Might be just me, but I always find it irritating to see a space between error codes. Both for pylint and mypy. Similarly in the other test files.

Reply:
Oh wow, I actually thought the preferred style was to separate them. I'm fine either way but thought that was the style we were going for.

Reply:
Yeah I  try to separate them myself :smile: (like arguments in a function call)

Reply:
To be fair I've seen (and used) both 😄 For files I write myself, I do prefer without spaces. However if I'm just editing a random test file, I usually continue with the style that is present.